compiler-plugin/src/main/resources/diagnostics.properties

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2025, WSO2 LLC. (http://www.wso2.org).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+error.non.json.expected.type.not.yet.supported=\
+  subtypes of ''anydata'' that are not subtypes of ''json'' are not yet supported as the expected type for {0}
+
+error.unexpected.arguments=\
+  unexpected arguments: expected ''1'' argument, found ''{0}''
+
+error.expected.a.subtype.of.np.model=\
+  incompatible expression: expected ''ballerina/np:Model'', found ''{0}''
```

Comment:
```suggestion
  incompatible expression: expected sub type of ''ballerina/np:Model'', found ''{0}''
```
wdyt?

Reply:
Kept it consistent with compiler errors.